packages/calling/src/CallingClient/line/index.ts

```diff
@@ -69,6 +69,8 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
 
   private callManager: ICallManager;
 
+  private serviceData: CallingClientConfig['serviceData'];
```

Comment:
Needed to create this local variable so we can check for guest calling enum.
Did not see any other way - open to comments tho.